### PR TITLE
[fix](vertical compaction) fix uint32_t init value

### PR DESCRIPTION
**Pull request by yixiutt**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/vec/olap/vertical_merge_iterator.h [hunk 1]
```diff
@@ -190,14 +190,14 @@ class VerticalMergeIteratorContext {
     size_t _ori_return_cols = 0;
 
     // segment order, used to compare key
-    uint32_t _order = -1;
+    uint32_t _order = 0;
 
-    uint32_t _seq_col_idx = -1;
```
Comment (zhannngchen) [inline be/src/vec/olap/vertical_merge_iterator.h:195]: `_seq_col_idx` change to int32_t is better? align with the type in schema

Comment (zhannngchen) [inline be/src/vec/olap/vertical_merge_iterator.h:195]: `_seq_col_idx` change to int32_t is better? align with the type in schema

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.14 seconds
 load time: 484 seconds
 storage size: 17170631480 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230202164338_clickbench_pr_89719.html

File: be/src/vec/olap/vertical_merge_iterator.h (hunk 1 above)
Comment (yixiutt) [reply be/src/vec/olap/vertical_merge_iterator.h:195]: fixed

**Review by zhannngchen — APPROVED** [review]
LGTM

**Pull request merged**